RFC/src/RFC-0140 Syncing and seeding.md

```diff
@@ -0,0 +1,132 @@
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
+THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+## Language
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", 
+"NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in 
+[BCP 14](https://tools.ietf.org/html/bcp14) (covering RFC2119 and RFC8174) when, and only when, they appear in all capitals, as 
+shown here.
+
+## Disclaimer
+
+The purpose of this document and its content is for information purposes only and may be subject to change or update
+without notice.
+
+This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community regarding the
+technological merits of the potential system outlined herein.
+
+## Goals
+
+This document describes the process of Syncing, seeding, pruning and cut-through.
+
+## Related RFCs
+
+* [RFC-0110: Base Nodes](RFC-0110_BaseNodes.md)
+
+## Descriptions
+
+### Syncing
+
+When a new node comes online, loses connection or encounters a chain re-organisation that is longer than it can tolerate, it must enter syncing mode. This will allow it to recover its state to the newest up to date state. Syncing can be divided into 2 [SynchronisationStrategy]s, complete sync and sync. Complete sync will involve that the node communicates with an archive node to get the complete history of every single block from genesis block. Sync will involve the node getting every block from its [pruning horizon](pruninghorizon) to [current head](current head), as well as every block header from genesis block. 
+
+#### Complete Sync
+
+Complete sync is only available from archive nodes, as these will be the only nodes that will be able to supply the complete history required to sync every block with every transaction from genesis block up onto [current head](currenthead). 
+
+
+
+#### Syncing process
+
+The syncing process MUST be done in the following steps:
+
+1. Set its [SynchronisationState] to `Synchronising`.
+2. Load a bootstrap list of peers from a configuration file, or a cached list.
+3. For each peer in this list:
+   1. Establish a connection with the peer.
+   2. Request a peer list from that peer.
+   3. Request information about the most recent chain state (total accumulated work, block height, etc.) from the peer.
+4. Choose the longest chain based on POW. 
```

Comment:
POW = PoW? Just check what is we are using elsewhere in the docs for consistency but usually in an acronym like this conjunctions and propositions (i.e. of) would be lowercase.

Reply:
This was asked by one of the other reviews, but I think PoW is better.